### DISCUSSION TITLE
30F32C15-BCB4-EB75-45FE-B2945B1C7D83

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#165] [General]
# 678191 (2017 OF₆₉)

2017 OF₆₉ is a large trans-Neptunian object. It is a plutino, in a 2:3 resonance with Neptune.

https://celestia.mobi/addon?item=30F32C15-BCB4-EB75-45FE-B2945B1C7D83

**Comment by MichaelAggelostheoti** [comment]
Cool

**Comment by pm26041980-arch** [comment]
I think 678191 (2017 OF₆₉) got bugged
